### ISSUE TITLE
Remnants of plugin remain on user's computer, even after plugin uninstall

### ISSUE THREAD

**Issue by pm64** [darkreader/darkreader#5447]
# Bug Report

## Describe the bug
If Dark Reader is uninstalled, it is expected that all material related to the plug-in shall be removed from the user's computer. However, it has been observed that if the plugin is then reinstalled, even months later, settings associated with the prior installation are restored. The plugin should either be completely uninstalled during the uninstallation process, or user guidance on performing complete removal should be provided.

## Steps to reproduce
1. Install Dark Reader
2. Alter any settings such that they differ from default values
3. Uninstall Dark Reader
4. Reinstall Dark Reader
5. Observe that settings reflect values from the previous installation, not default values

## Expected behavior
Uninstallation should completely remove the plugin (including its settings). On reinstallation, the settings should therefore reflect default values.

## Actual behavior
Remnants of the plugin remain on the user's computer, causing previous settings to be restored, and making it impossible for the user to restore default settings.

## Screenshots
N/A

## System Information

- OS: Windows 10 x64
- Browser: Firefox Developer 88.0b6 (64-bit)
- Dark Reader Version: 4.9.30

## Additional context
N/A

**Comment by Gusted** [comment]
Hi @pm64!

Such behavior is caused by an user's sync preference. Which Dark Reader cannot remove on an uninstall, it is possible to use the reset feature within Dark Reader to remove settings. However the uninstall process is out of our reach to execute any code.

Regards,
Gusted

**Comment by Gitoffthelawn** [comment]
I agree with both pm64 and Gusted.  This is a known issue (well, at least to me and Gusted) in Firefox.  Uninstalling an extension does not always remove all of its data (even from the local profile), regardless of whether or not sync is completely disabled.  Best to report the issue on Bugzilla to Mozilla.  I really hope they fix it.

The only thing I can think that Dark Reader can do is detect it's own installation and notify the user on installation if any settings are not the default.  Dark Reader can then ask the user to reset all settings or use previous settings.

**Comment by pm64** [comment]
Hi @Gusted and @Gitoffthelawn, thank you for your kind responses. I agree that the root of the problem lies in Mozilla's design, and am happy to approach the solution from that side. But in the meantime, the impact on this add-on can still be mitigated through documentation. I was going to ask if there was a SQLite table I could truncate or otherwise affect to achieve the desired result. But @Gusted, you mention a "reset" feature. I recall seeing this option in the past, can you please refresh my memory as to where it can be found?

**Comment by Gusted** [comment]
> @Gusted, you mention a "reset" feature. I recall seeing this option in the past, can you please refresh my memory as to where it can be found?

To activate the new user interface do the following:

    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.

Reset button (click Settings, Manage settings).
